src/main/kotlin/io/github/rybalkinsd/kohttp/ext/RequestExt.kt

```diff
@@ -0,0 +1,45 @@
+package io.github.rybalkinsd.kohttp.ext
+
+import okhttp3.Headers
+import okhttp3.Request
+import okhttp3.RequestBody
+import okio.Buffer
+import java.nio.charset.Charset
+
+/**
+ *  Build curl command of the request
+ *
+ * @author doyaaaaaken
+ */
+internal fun Request.buildCurlCommand(): String {
+    return buildString {
+        append("curl -X ${method()}")
+        append(buildCurlHeaderOption(headers()))
+        append(buildCurlBodyOption(body()))
+        append(" \"${url()}\"")
+    }
+}
+
+private fun buildCurlHeaderOption(headers: Headers): String {
+    return headers.asSequence().map { header ->
```

Comment:
Btw, `Header` as a type alias could be "decoupled", see below
```suggestion
    return headers.asSequence().map { (name, value) ->
```